### ISSUE TITLE
WWE 2K22 (1255630)

### ISSUE THREAD

**Issue by JGussey** [ValveSoftware/Proton#5737]
# Compatibility Report
- Name of the game with compatibility issues:  WWE 2k22
- Steam AppID of the game: 1255630

## System Information
- GPU:  AMD RX 6700 XT red devil
- Driver/LLVM version:   [AMD/ATI] Navi 22 [Radeon RX 6700/6700 XT / 6800M] (rev c1)
- Kernel version:   5.16.18-200.fc35.x86_64 
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/JGussey/421f27f8c8e1271ab8a98d25f1cfa03c
- Proton version: I've tried Experimental/GE/ and 7.0

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

https://pastebin.com/AKhKtMVw

## Symptoms 
When clicking play, game launches then instantly crashes. Their is reports on protondb of some people getting to work, but what they have tried has not worked for myself or many others.

## Reproduction

I apologize if I messed anything up here. First time reporting a issue and wanted to get it up there.

**Comment by Blisto91** [comment]
It would be interesting to see if the behavior is the same on bleeding edge experimental so the latest `vkd3d-proton` version can be tested. 
Tho for that to work you need to be using at least mesa (radv) 22.0.

As a extra note if it was `GE-Proton7-10` you were trying to use that one should also need mesa 22.0. GE-Proton7-9 should be fine with your current driver tho, but i don't expect that to make the game work.

**Comment by Nerellus** [comment]
I've had no luck launching it with the RADV driver. It works fine with the open AMDVLK driver. It launches with the -PRO driver as well, but misrenders in matches.

**Comment by Blisto91** [comment]
@Nerellus Which RADV version were you using? And which proton version

**Comment by Nerellus** [comment]
No particular version. I've used custom builds of the main dev branch since the game released. Proton Experimental/BE.

**Comment by Blisto91** [comment]
mesa-git/dev should work with bleeding edge yes.

If you suspect the RADV driver itself and not `vkd3d-proton` or proton can you make a mesa issue?

**Comment by Nerellus** [comment]
Sorry, I guess I was unclear. I was never suggesting a proton/vkd3d-proton issue. I was offering advice on which driver to use to get it to work. I haven't been able to get any useful info to open a mesa issue yet, but I'll continue my efforts unless someone beats me to it.

**Comment by Blisto91** [comment]
Ye i understood :D
I do not own the game myself so i'm probably not the one beating you to it hehe.

**Comment by selektionsrest** [comment]
It's the same with Mesa 22 & bleeding-edge Experimental.

**Comment by Blisto91** [comment]
I would imagine since it doesn't work with mesa and some of the reports on protondb is from the steam deck that it will get some priority.
Tho a `vkd3d-proton` or `mesa` issue needs to be opened. Maybe more likely the latter if it works with `amdvlk` and the pro driver. Tho i'm not entirely sure on that one

@selektionsrest can you post a proton log from bleeding edge?

**Comment by JGussey** [comment]
I’m really sorry guys I’ve been dealing with a lot of crap and haven’t had a chance to check or mot sure what to do next

**Comment by Blisto91** [comment]
It's okay. You have no responsibility to do anything here. It's just good you opened a issue :D

**Comment by selektionsrest** [comment]
[steam-1255630.log](https://github.com/ValveSoftware/Proton/files/8405255/steam-1255630.log)
 I hope this is helpful

**Comment by JGussey** [comment]
> It's okay. You have no responsibility to do anything here. It's just good you opened a issue :D

Okay cool. I really hope we can find a fix for it. I love the game, they did so good with it this time and I've fallen in love with fedora so much.

**Comment by Blisto91** [comment]
> [steam-1255630.log](https://github.com/ValveSoftware/Proton/files/8405255/steam-1255630.log) I hope this is helpful

At least to my newb eye i'm not seeing anything detailed of interest in the log.
Gonna see how to get more debug information out of `vkd3d-proton` when i get home today. I think you need a non release build for it.

**Comment by selektionsrest** [comment]
Tested a bit yesterday. With AMDVLK + Proton Experimental it runs great if you start it windowed.

**Comment by JGussey** [comment]
Hmm I tired just now with that and It still wont start. I figure I'll just be waiting till valve messes with it.

**Comment by Blisto91** [comment]
I haven't had time to look at anything yet. But i'll try to get the game maybe and see to make a issue somewhere. (Not a valve dev btw)

**Comment by Nerellus** [comment]
For what it's worth, I'm no longer able to reproduce. This is now working with RADV for me - I updated both RADV and Proton Experimental (BE) at roughly the same time, so not sure which landed a fix.

All is working well here with an RX 570 now.

**Comment by McMarius11** [comment]
maybe this fixes the game for you

digita1hound:
Properties >general>launch options add: -force-dx11 Compatibility checked and proton experimental is selected.

https://www.reddit.com/r/SteamDeck/comments/u7hasq/wwe2k22_after_109_update_proton_experimental/
https://www.reddit.com/r/SteamDeck/comments/u7mlj8/wwe2k22_proton_experimental_on_high_settings_and/

**Comment by JGussey** [comment]
So its "working" right now for me. But its like the matches are played in x10 speed cheat code on and after a bit the game crashes i would take from running so fast.

**Comment by Nerellus** [comment]
> So its "working" right now for me. But its like the matches are played in x10 speed cheat code on and after a bit the game crashes i would take from running so fast.

That's because of a new game option. Options → Extras → Graphics Settings → Timed Game Update. Set to Yes for gameplay to get back to normal speed.

**Comment by Grandchado** [comment]
it seems like the issue is with the online part of the game. because when i play the game without internet, no crashes occur. tested with proton experimental.

**Comment by TheAceThatRunsThePlace** [comment]
Yeah, can confirm that it seems as if some part of the internet connectivity is causing the issue. The game is pretty much flawless while in offline mode, but then that cuts off a lot of essential features of the game as you aren't connected.

If that were fixed the game would be perfect.
